app/components/album-art.js

```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+import layout from '../templates/components/album-art';
+
+export default Ember.Component.extend({
+  layout: layout,
+  classNames: ["album-art"],
+  classNameBinding: ["loading"],
```

Comment:
I re-based on top of master and couldn't reproduce. Maybe it was some thing to do with the try-catch block?